.github/CONTRIBUTING.md

```diff
@@ -1,36 +1,169 @@
-# Contributing to BeEF
+# Contributing
+### Anyone is welcome to make BeEF better!
+Thank you for wanting to contribute to BeEF. It's effort like yours that helps make BeEF such a great tool.
 
-Anyone is welcome to make BeEF better!
+Following these guidelines shows that you respect the time of the developers developing this open source project and helps them help you. In response to this, they should return that respect in addressing your issue, assisting with changes, and helping you finalize your pull requests.
 
-Below are the steps needed to add code to BeEF on Github to the master branch:
+###  We want any form helpful of contributions!
```

Comment:
The `We want any form helpful of contributions!` section is duplicated. Also, it should be `We want any form of helpful contributions!`

Reply:
noted, thanks for the catch! I'll Update in the next push.